### PR TITLE
Added installation guide for Gentoo Linux

### PR DESCRIPTION
**Pull request by renich**
Not much to say; the title is self-explanatory.

### CODE REVIEW

File: installation/on_gentoo_linux.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# On Arch Linux
```
Comment (jhass) [inline installation/on_gentoo_linux.md]: Copy paste error ;)

**Comment by renich** [comment]
Ah, sorry. Fixed it just now.

**Comment by mverzilli** [comment]
Thank you!

**Comment by renich** [comment]
:D. No problem! Thanks for the merge.

**Pull request merged**